packages/design-system/src/components/Form/Primitives/Select/Select.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { forwardRef, ReactElement, Ref, SelectHTMLAttributes } from 'react';
+import classnames from 'classnames';
+import InputWrapper, { AffixesProps } from '../InputWrapper/InputWrapper';
+
+type SelectProps = Omit<SelectHTMLAttributes<any>, 'prefix'> & {
+	children: ReactElement[];
+	readOnly?: boolean;
+} & AffixesProps;
+
+import styles from './Select.module.scss';
+import { Icon } from '../../../Icon/Icon';
+
+const Select = forwardRef((props: SelectProps, ref: Ref<HTMLSelectElement>) => {
+	const {
+		children,
+		className,
+		prefix,
+		suffix,
+		placeholder,
+		required,
+		readOnly = false,
+		disabled = false,
+		...rest
+	} = props;
+	return (
+		<InputWrapper prefix={prefix} suffix={suffix} readOnly={!!readOnly} disabled={!!disabled}>
+			<div className={styles.select__wrapper}>
+				<select
+					{...rest}
+					disabled={disabled || readOnly}
```

Comment:
Same comment for disabled here

Reply:
Sure, well spotted, there's no such thing as a readOnly select anyways (it's an input).